test_conformance/integer_ops/test_intmad24.cpp

```diff
@@ -196,13 +193,13 @@ int test_integer_mad24(cl_device_id device, cl_context context, cl_command_queue
     cl_kernel kernel[2*NUM_PROGRAMS];
     size_t threads[1];
 
-    int                num_elements;
+    int n_elems;
```

Comment:
Minor: I think we could reduce diffs and eliminate some possible confusion about the difference between `n_elems` and `num_elements` by removing `n_elems` and assigning to the `num_elements` function argument instead.